ai_news_generator/src/ai_news_flow/main.py

```
@@ -0,0 +1,294 @@
+            "completed": self.state.generation_completed,
+            "research_sources": len(self.state.research_report.research_data) if self.state.research_report else 0,
+            "word_count": self.state.final_article.word_count if self.state.final_article else 0,
+            "processing_time": self.state.processing_duration,
+            "readability_score": self.state.edited_content.readability_score if self.state.edited_content else 0
+        }
+
+
+def kickoff(topic: str, temperature: float = 0.7, max_sources: int = 10):
+    """Convenience function to run the complete news generation flow."""
+    flow = NewsGeneratorFlow(topic=topic, temperature=temperature, max_sources=max_sources)
+    result = flow.kickoff()
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)